src/patternfly/components/Accordion/docs/code.md

```diff
@@ -1,5 +1,11 @@
 ## Overview
 
+The accordion component can be built in two different ways. 
+The default way uses `<div>` and `<h3>` tags to build the component. 
```

Comment:
Related to my previous comment, I'm not sure that we want to label anything in core as a default, since the concept of defaults doesn't really exist here the way they do in the react repo. In react, there are defaults based on empty props, and for this component, I think the default variation would be the definition list variation.

Here, I would just say that one variation uses definition list and another uses headings. And also that we are using `<h3>` but really the heading levels that you use should fit within the rest of the heading outline on your page.